robocheck-core.py

```diff
@@ -75,15 +75,7 @@ def main():
         return
 
     language = configuration.getLanguage()
-    penaltyFlag = configuration.getPenaltyFlag()
-
-    if penaltyFlag is True:
-        penaltyDictionary = configuration.getPenaltyDictionary()
-        errorsToLookFor = penaltyhandler.getErrorsToLookFor(penaltyDictionary)
-    else:
-        errorsToLookFor = configuration.getErrorsToLookFor()
-
-
+    errorsToLookFor = configuration.getErrorsToLookFor()
```

Comment:
You moved the code for loading the penalties to the configuration handler, right?